chef_master/source/custom_resources.rst

```diff
@@ -23,6 +17,12 @@ For example, Chef includes built-in resources to manage files, packages, templat
 
 .. end_tag
 
+Changed in Chef Client 12.5. This is the recommended approach for all custom resources starting with Chef client 12.5. If you are using an older version of Chef client, please use the version picker (in the top left of the navigation) to select your version, and then choose the same topic from the navigation tree ("Extend Chef > Custom Resources"). See also https://github.com/chef-cookbooks/compat_resource for using custom resources with chef-client 12.1 - 12.4.
```

Comment:
We no longer have a version picker in our left nav, so I'd just strike this sentence.